internal/tui/model_list.go

```diff
@@ -138,20 +171,73 @@ type errorMsg struct {
 	Err error
 }
 
+type keyMap struct {
+	Save  key.Binding
+	Enter key.Binding
+	Quit  key.Binding
+}
+
+func (k keyMap) ShortHelp() []key.Binding {
+	return []key.Binding{k.Save, k.Enter, k.Quit}
+}
+
+func (k keyMap) FullHelp() [][]key.Binding {
+	return [][]key.Binding{
+		{k.Save, k.Enter, k.Quit},
+	}
+}
+
+var inputKeys = keyMap{
+	Save: key.NewBinding(
+		key.WithKeys("s", "enter"),
+		key.WithHelp("enter/ctrl+s", "save suggest"),
+	),
+	Quit: key.NewBinding(
+		key.WithKeys("esc", "esc", "ctrl+c"),
+		key.WithHelp("esc/ctrl+c", "quit"),
+	),
+}
```

Comment:
I think for consistency it is best to use the same keymap system throughout. So then, how would you feel about refactoring out of this `keyMap` struct to be of type `KeyMap`?

This way, there would be an `EditKeyMap()` function on the model, analogous to the current `KeyMap()` function. 

One benefit, for example, you wouldn't have to define your own `ShortHelp` and `FullHelp` functions. There is also no more duplication of keyboard shortcuts.